reference/webhooks.v3.yml

```diff
@@ -1,463 +1,644 @@
 openapi: 3.0.0
 info:
-  title: Webhooks v3
+  title: Webhooks
+  description: |
+    Create and manage webhooks to get notified when specific events occur on stores. For more information, see [Webhooks Overview](https://developer.bigcommerce.com/api-docs/store-management/webhooks/overview).
+
+
+     <div class="HubBlock--callout">
+     <div class="CalloutBlock--info">
+     <div class="HubBlock-content">
+
+    > ### Note
+    > * Webhooks endpoints are available on both the V2 and V3 REST API.

```

Comment:
We should take into account that V2 & V3 endpoints are a bit different. For example, `getAllWebhooks` endpoint for v2 doesn't support pagination. Also, in v2 response json is not wrapped in `data` parent field. So it might have sense to differentiate it until v2 is not fully removed.